src/pages/ListPage/PolicyTableError.ts

```diff
@@ -17,26 +19,30 @@ export const policyTableError = (canReadAll: boolean, requestHasError?: boolean,
             case 404:
                 return {
                     icon: ExclamationCircleIcon,
+                    iconColor: GlobalDangerColor200,
```

Comment:
I think the red exclamation mark is too aggressive.

Reply:
Maybe, I copied from: https://patternfly-react.surge.sh/documentation/react/demos/tableemptystatesdemo#error